index.js

```diff
@@ -304,20 +304,31 @@ export const Classes = {
 		selected: 'selected'
 	},
 	outcomes: {
+		assessed: 'assessed',
+		demonstration: 'demonstration',
+		demonstratableLevel: 'demonstratable-level',
 		intent: 'intent',
 		intents: 'intents',
 		intentList: 'intent-list',
+		levelOfAchievement: 'level-of-achievment',
 		organizationIntentList: 'organization-intent-list',
 		outcome: 'outcome',
 		outcomes: 'outcomes',
 		selected: 'selected',
-		suggested: 'suggested',
-		demonstratableLevel: 'demonstratable-level'
+		suggested: 'suggested'
 	},
 	meetings: {
 		bookable: 'bookable'
 	},
-	relativeUri: 'relative-uri'
+	relativeUri: 'relative-uri',
+	userProgress: {
+		outcomes: {
+			root: 'user-progress-outcomes',
```

Comment:
I think changing from `root` to `outcomes` would be more in line with the existing similar implementations (including the general case `outcomes.outcomes` from above).

```suggestion
			outcomes: 'user-progress-outcomes',
```